src/test/java/org/zalando/nakadi/utils/RandomSubscriptionBuilder.java

```diff
@@ -83,6 +83,7 @@ public Subscription build() {
         final Subscription subscription = new Subscription();
         subscription.setId(id);
         subscription.setCreatedAt(createdAt);
+        subscription.setUpdatedAt(createdAt);
```

Comment:
and here as well. It introduces some hidden trick which may bring problems in the future.